libandroid-navigation-ui/src/main/res/values/strings.xml

```diff
@@ -60,4 +60,8 @@
             translatable="false">navigation_view_instance_state</string>
     <string name="navigation_running"
             translatable="false">navigation_running</string>
+    <string name="mapbox_navigation_route_line_cap_round"
```

Comment:
After https://github.com/mapbox/mapbox-navigation-android/pull/1818#issuecomment-483373643 this is not needed anymore, right?